README.md

```diff
@@ -13,6 +13,33 @@ Import on SBT:
 
     "com.codacy" %% "bitbucket-scala-client" % "<VERSION>"
 
+To build the `BitbucketClient` you need to pass a builder function from Unit => WSClient
+
+#### Play 2.4
+
+```Scala
```

Comment:
We should have this information in a CHANGELOG.md so people can be aware what are the necesssary changes when we bump the major version

Reply:
👍